decidim-admin/app/controllers/decidim/admin/concerns/participatory_process_admin.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module Concerns
+      # This concern is meant to be included in all controllers that are scoped
+      # into a participatory process' admin panel. It will override the layout
+      # so it shows the sidebar, preload the participatory process, etc.
+      module ParticipatoryProcessAdmin
+        extend ActiveSupport::Concern
+
+        included do
+          helper_method :participatory_process
+          before_filter :ensure_participatory_process
```

Comment:
I think in Rails 5 you should favor `before_action` since `before_filter` is deprecated.